content/en/docs/Tasks/make-index-available-on-cluster.md

```diff
@@ -27,6 +27,11 @@ spec:
       interval: 10m
 ```
 
+### Catalog scoping
+By default, a CatalogSource is scoped to the namespace in which it is created in. For example, a CatalogSource in namespace `A` can only be used in dependency resolution for subscriptions that originate in namespace `A` as well. A subscription in another namespace that references the CatalogSource in `A` will not be able to use the catalog contents for dependency resolution. This provides some basic multitenancy: CatalogSource in one namespace do not interfere with CatalogSource in another. 
```

Comment:
Is the cognitive load needed to grasp the phrase `dependency resolution` necessary for this section of the doc? How about rephrasing it to something like "operators in a CatalogSource in namespace `A` can only be subscribed to from namespace `A`, i.e a subscription created in namespace `B` for an operator from the CatalogSource in namespace `A` will fail"?

Reply:
I think I'd largely agree here with Anik - it seems sufficient to me to explain that CatalogSource(es) are scoped in the namespace they've been created in, and in order to install a namespaced-scoped Operator that's defined in that CatalogSource, you must also create that Subscription in the same namespace as the CatalogSource definition. The only caveat being the `olm` namespace acting as a global namespace for dependency resolution.